scripts/calculateRevenue/protocols/beefy/index.ts

```diff
@@ -45,37 +49,77 @@ export function getVaultTvl(
   return lastTvl
 }
 
+/**
+ * Given a list of historical token prices and a timestamp, returns the nearest token price
+ * to the specified time.
+ * Requires the list of tokenPrices to be sorted from earliest to most-recent.
+ */
+export function getTokenPrice(
```

Comment:
how accurate does this need to be? Since we're using the first price after the given timestamp, if there's a large gap in the data, we might be off by a bit.

Reply:
I believe our endpoint (at least as far as I saw) returns hourly prices. It's not perfect granularity, but I think it will probably suffice, especially for this v0.